Client/Frontend/Browser/New Tab Page/NewTabPageViewController.swift

```diff
@@ -109,6 +109,9 @@ class NewTabPageViewController: UIViewController {
   private var preventReloadOnBraveNewsEnabledChange = false
 
   private let notifications: NewTabPageNotifications
+  
+  /// Internal url to access the new tab page.
+  static let ntpInteralURL = URL(string: "\(InternalURL.baseUrl)/\(AboutHomeHandler.path)#panel=0")
```

Comment:
Just a personal preference but I would use a ! here. Everything is hard coded here. It either fails or it doesn't

Reply:
I never use force unwraps even for impossible cases :o